e2e/testcases/helm_sync_test.go

```diff
@@ -98,6 +100,13 @@ func TestPublicHelm(t *testing.T) {
 
 	nt.T.Log("Update RootSync to sync from a public Helm Chart without specified release namespace")
 	nt.MustMergePatch(rs, `{"spec": {"helm": {"namespace": ""}}}`)
+	nt.T.Cleanup(func() {
+		// WordPress MariaDB creates a PVC in the namespace, which is left after the test.
+		// The cleanup function deletes the wordpress-mariadb PVC so that MariaDB can be reinstalled in the namespace in the next test.
+		if err := deleteWordpressPVC(nt); err != nil {
```

Comment:
Is there a `PVC` created in the default namespace as well by the updated RootSync on line `102`?

Reply:
Yes, and the `deleteWordpressPVC` function deletes the PVC in the `default` namespace. There is also a PVC created in the `wordpress` namespace from `root-sync-helm-chart-cr.yaml` that is applied in line 61. That is deleted as a side-effect of pruning the `wordpress` namespace.

Reply:
Ah I didn't realize `DefaultHelmReleaseNamespace` was defined as `default`. Ack, thank you for explaining